proj-xcode/Classes/sdk/PowerAuthSDK.m

```diff
@@ -575,9 +575,10 @@ - (PA2OperationTask*) createActivationWithName:(NSString*)name
 				}
 			} else {
 				// Activation error occurred
-				errorToReport = [NSError errorWithDomain:PA2ErrorDomain code:PA2ErrorCodeInvalidActivationData userInfo:nil];
+                NSDictionary *encryptedResponseDictionary = [NSJSONSerialization JSONObjectWithData:httpData options:kNilOptions error:nil];
+				errorToReport = [NSError errorWithDomain:PA2ErrorDomain code:PA2ErrorCodeInvalidActivationData userInfo:encryptedResponseDictionary];
```

Comment:
A nullability of `httpData` should be tested, otherwise it might a crash the application on `"NSInvalidArgumentException", "data parameter is nil"` in that JSON deserialization.